src/main/java/com/three_tech_solutions/slot_app/services/implementations/StudentServiceImpl.java

```diff
@@ -144,8 +146,14 @@ public StudentResponse updateStudent(UUID studentId, UpdateStudentRequest studen
     }
 
     @Override
-    public Page<Student> getStudentsByUserAndNameAndLastNameAndDni(User user, String filters, Pageable pageable) {
-        return studentRepository.getStudentsByUserAndNameAndLastnameAndDni(user, filters, pageable);
+    public Page<Student> getStudentsByUserAndNameAndLastNameAndDni(User user, String filters,String orderBy, String orderDirection,Pageable pageable) {
+        orderDirection = verifyOrderParams(orderBy, orderDirection);
+
+        Sort sort = createSortIfIsValid(orderBy, orderDirection);
+
+        Pageable pageableWithSort = createPageableWithSort(pageable, sort);
+
+        return studentRepository.getStudentsByUserAndNameAndLastnameAndDni(user, filters, pageableWithSort);
```

Comment:
Ahora esto ya no necesitarias crear el `pageable` aca, sino que podes hacer lo siguiente


```suggestion
        return studentRepository.getStudentsByUserAndNameAndLastnameAndDni(user, filters, pageable);
```